From 6ee13bdab2654c455ddf34b61825bbeb44ae9844 Mon Sep 17 00:00:00 2001
From: Abhishek <apatgaonkar@habana.ai>
Date: Tue, 08 Oct 2024 06:09:40 +0300
Subject: [PATCH] Revert "[FSW-419] fs1: rma: add rma_manifest field"

This reverts commit 993c9cc3ac15f015b8935af16eeebf49a4e7673f.
Rma manifest bit is removed from manifest. Only rma efuse
bit will be checked to enable rma during boot and any
valid manifest can be booted in rma.

Change-Id: I7e7524a18c352304217956639bac5bdc64f1326a
---

diff --git a/board/habanalabs/fs1_arc/post-image-spi.sh b/board/habanalabs/fs1_arc/post-image-spi.sh
index fbc0c8a..1ecb838 100755
--- a/board/habanalabs/fs1_arc/post-image-spi.sh
+++ b/board/habanalabs/fs1_arc/post-image-spi.sh
@@ -571,12 +571,9 @@
 	#Prepare key index
 	printf "%02x" $key_idx | rev_bytes_stdin | xxd -r -p -s +25 - $img_file
 
-	local rma_manifest=0
-	printf "%02x" $rma_manifest | rev_bytes_stdin | xxd -r -p -s +26 - $img_file
-
 	local rsvd_data=0
 	#Prepare reserved data
-	printf "%02x" $rsvd_data | rev_bytes_stdin | xxd -r -p -s +27 - $img_file
+	printf "%04x" $rsvd_data | rev_bytes_stdin | xxd -r -p -s +26 - $img_file
 
 	local offset_array=$IMG_PTR_SZ
 	#Prepare security version for Stable image
